notion/CHANGELOG.md

```diff
@@ -2,7 +2,5 @@
 
 ## 1.0.0 / 2023-05-31
 
-***Added***:
-
-* Initial Release
+* [Added] Initial Release
```

Comment:
The Changelog format must adhere to the newer standards to provide a stronger, consistent UX across integration offerings. Could you please undo this change?

Reply:
done ✅

Reply:
Sorry, I think my comment here may have been confusing. Could you ensure it looks like this:

```
***Added***:

* Initial Release
```

The full CHANGELOG style guide can be found here - https://datadoghq.dev/integrations-core/guidelines/pr/
And here's a proper example of what it needs to look like - https://github.com/DataDog/integrations-extras/blob/master/ns1/CHANGELOG.md

That should get CI all green again 🙂

Reply:
Added the change, please let me know if that looks correct

Reply:
Thanks, I think the formatter is looking for a single newline at the end of the file, with no whitespaces on that line.

Reply:
Nice, looks like that did the trick!